Extensions/IISWebAppDeploy/Src/Tasks/SqlDacpacDeploy/DeployToSqlServer.ps1

```diff
@@ -236,4 +240,10 @@ function Main
     }
 
     RunRemoteDeployment @remoteDeploymentArgs
+}
+
+function ConvertTo-Json20([object] $item){
```

Comment:
This is already in the utility that you are importing. Why do you need this again?